### PR TITLE
Added ability to upload experimental bags

### PR DESCRIPTION
**Pull request by pato**
Made the parameter parsing more rich by allowing the ability to also specify whether you want experimental bag files (of the form extra_*.bag) to be compressed and uploaded as well as outlined in issue #2

### CODE REVIEW

**Comment by jack-oquin** [comment]
Here again, all I think we really need is to add a parameter for the bag file prefix. Then, we can just run the command once for each desired prefix (`bwi_`, `extras_`).

**Comment by pato** [comment]
Rewrote it to take a parameter for the prefix (or default to bwi).
Take a look @jack-oquin  and tell me what you think

File: bwilab_scripts/scripts/upload [hunk 1]
```diff
@@ -2,56 +2,65 @@
...
+    esac
+done
+
+suffix="_*.bag"
+glob=$prefix$suffix
+
+# Ensure the hostname is set
 if [ "$HOSTNAME" == "" ]
-then    echo '$HOSTNAME must be set for this operation'
+then    echo "$HOSTNAME must be set for this operation"
```
Comment (jack-oquin) [inline bwilab_scripts/scripts/upload]: This change causes the shell to attempt to resolve `$HOSTNAME`, which is not defined. Using single quotes prints $HOSTNAME as part of the message.

File: bwilab_scripts/scripts/upload [hunk 2]
```diff
@@ -2,56 +2,65 @@
...
+        echo "By default, will delete the files after uploading"
         echo ""
-        echo "Supports at most one of these options:"
-        echo -e "\t-d\tdelete files uploaded successfully (default)"
-        echo -e "\t-h\tprint this help message"
-        echo -e "\t-k\tkeep uploaded files"
+        echo "Supports these options:"
+        echo -e "\t-h [--help]\tprint this help message"
+        echo -e "\t-p\tspecify the bag prefix (will upload nodes matching 'PREFIX_*.bag'"
+        echo -e "\t-e\tkeep the files after uploading"
```
Comment (jack-oquin) [inline bwilab_scripts/scripts/upload]: I think there's a typo: the keep option should be `-k`, not `-e`.

**Comment by jack-oquin** [comment]
Other than two trivial problems, it looks good to me. 

Thanks for working on this, Pato. I'd like to merge it before doing some further improvements to the `upload` script.

**Comment by jack-oquin** [comment]
See #4 for some further changes I'd like to make after this PR is merged.

**Comment by pato** [comment]
I've made all the changes you pointed out (also realized i was still using a hardcoded `bwi_*.bag` glob match, so I fixed that. Should be good to go now! Thanks for the review

**Comment by jack-oquin** [comment]
Looks good, thanks. 

Glad you caught that other glob. That bug would have been embarrassing, :smile:

**Pull request merged**